cmake/sip_helper.cmake

```diff
@@ -5,9 +5,7 @@ set(__PYTHON_QT_BINDING_SIP_HELPER_INCLUDED TRUE)
 
 set(__PYTHON_QT_BINDING_SIP_HELPER_DIR ${CMAKE_CURRENT_LIST_DIR})
 
-set(Python_ADDITIONAL_VERSIONS "${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}")
-find_package(PythonInterp "${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}" REQUIRED)
-find_package(PythonLibs "${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}" REQUIRED)
```

Comment:
I see `PYTHON_INCLUDE_DIRS` used below, but that [variable comes from `FindPythonLibs`](https://cmake.org/cmake/help/latest/module/FindPythonLibs.html). Any use of that must be replaced. A target dependency on [one of `FindPython3`'s imported targets](https://cmake.org/cmake/help/latest/module/FindPython3.html#imported-targets), probably `Python3::Module`, is usually the best option. It may be easier to use the [result variable `Python3_INCLUDE_DIRS`](https://cmake.org/cmake/help/latest/module/FindPython3.html#result-variables) in this case.

Reply:
I replace `PYTHON_INCLUDE_DIRS` to `Python3_INCLUDE_DIRS` directly. `Python3_INCLUDE_DIRS` is provided by find_package (Python3 COMPONENTS Development)